runtime/src/comm/ofi/README.md

```diff
@@ -529,6 +529,11 @@ the same memory address occur in program order. Note that this is a stronger
 guarantee than is necessary to enforce the MCM because it ensures that all
 writes by a task occur in program order, independent of memory address.
 
+To force prior operations to be visible requires the FI_DELIVERY_COMPLETE flag
+in addition to the FI_FENCE flag. Otherwise, the FI_FENCE merely causes the
```

Comment:
I suggest  "To force prior operations to be visible, the FI_DELIVERY_COMPLETE flag is required in addition to the FI_FENCE flag."